app/views/devise/registrations/new.html.erb

```diff
@@ -3,14 +3,13 @@
 
   <%= bootstrap_form_for resource, as: resource_name, url: registration_path(resource_name) do |f| %>
     <%= devise_error_messages! %>
-    
+
     <%= f.email_field :email, autofocus: true %>
     <%= f.text_field :name %>
-    <%= f.password_field :password, autocomplete: "off", label: "Password (#{@minimum_password_length || 8} characters minimum)" %>
-    <%= f.password_field :password_confirmation, autocomplete: "off" %>
+    <%= f.password_field :password, label: "Password (#{@minimum_password_length || 8} characters minimum)" %>
```

Comment:
@CamLatimer same deal, can you revert these changes so the `autocomplete=off` comes back?